frontend/src/store/modules/shoots/getters.js

```diff
@@ -206,11 +220,37 @@ export default {
     }
   },
   sortItems (state, getters, rootState, rootGetters) {
-    return (items, sortByArr, sortDescArr) => {
+    let sortedUIDsAtFreeze
+    let sortByArrAtFreeze
+    let sortDescArrAtFreeze
+
+    const sortFreezedItems = (itemsToSort) => {
```

Comment:
I renamed the function to `orderFreezedItemsBySortedUIDsAtFreeze` because this is what the function actually does.
Moreover, I moved the logic that picks the item from the store <> freezedShootSkeletons to the store.